### ISSUE TITLE
Is this project abandoned? 

### ISSUE THREAD

**Issue by suhaibmujahid** [vektah/dataloaden#50]
I see no recent commits. Is it reliable to use this project in production?

**Comment by frederikhors** [comment]
@suhaibmujahid yes it is. No problem at all. Is not abandoned. @vektah is amazing!

**Comment by frederikhors** [comment]
@vektah are you still with us? Please answer, yes or not.

**Comment by microo8** [comment]
I think, that with the new 1.18 release there is no need for this tool
I've created a universal Dataloader using generics.
https://git.sr.ht/~ghost08/dataloader/tree/master

Although I don't know what to do with the copying of the value: https://git.sr.ht/~ghost08/dataloader/tree/master/item/dataloader.go#L149

There's also an example: https://git.sr.ht/~ghost08/dataloader/tree/master/item/dataloader_test.go

**Comment by microo8** [comment]
Oh that's great :) but will there be generics support? https://github.com/graph-gophers/dataloader/issues/83

**Comment by frederikhors** [comment]
It's OSS. Everything is uncertain. 😄

**Comment by vikstrous2** [comment]
@microo8  I did the same :) https://github.com/vikstrous/dataloadgen
